sqlglot/dialects/snowflake.py

```diff
@@ -805,6 +806,13 @@ def _parse_show_snowflake(self, this: str) -> exp.Show:
                 },
             )
 
+        def _parse_put(self) -> exp.Put:
+            self._match(TokenType.PUT)
+            source = self._parse_location_path()
+            target = self._parse_location_path()
+            props = self._parse_properties()
+            return self.expression(exp.Put, this=source, target=target, properties=props)
```

Comment:
Let's make sure that we'll fall back to producing a `Command` if the token following `PUT` is *not* a string. We don't wanna break existing `PUT` statements.